lib/collection/property-base.js

```diff
@@ -1,5 +1,6 @@
 var _ = require('../util').lodash,
     Description = require('./description').Description,
+    PropertyList = require('./property-list').PropertyList,
```

Comment:
Use flags in prototype?
On Thu, 10 Mar 2016 at 13:40, Aniket Panse <notifications@github.com> wrote:

> In lib/collection/property-base.js
> <https://github.com/postmanlabs/postman-collection/pull/6#discussion_r55645245>
> :
>
> > @@ -1,5 +1,6 @@
> >  var _ = require('../util').lodash,
> >      Description = require('./description').Description,
> > +    PropertyList = require('./property-list').PropertyList,
>
> Actually it is only needed because we internally have plural names for
> lists, but the collection format has singular names. [headers vs header,
> items vs item, etc]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/postmanlabs/postman-collection/pull/6/files#r55645245>
> .
>
-- 
--
Shamasis Bhattacharya

VPE / Postman